### DISCUSSION TITLE
Vsync not capping FPS

### DISCUSSION THREAD

**Discussion by Neekodesu** [ppy/osu#26359] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When using vsync mode instead unlimited, it's capping my FPS to 660-661 FPS (my monitor has 165hz), instead capping it at 165

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/47283957/c43b5d31-413b-488d-af92-a7ef2b3902b4)

### Version

Osu! Lazer app-2023.1231.0

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13817848/compressed-logs.zip)

**Comment by peppy** [comment, marked as answer]
Check your graphics drivers. You are overriding vsyhc somewhere.

**Reply by Neekodesu** [reply]
Is there any way to cap fps to your monitor refresh rate without forcing vsync? Like, instead x2/x4 refresh rate cap options, also add a x1 one

**Reply by peppy** [reply]
we may add this in the future.
